### ISSUE TITLE
Changing credentials on AWS scan results in error

### ISSUE THREAD

**Issue by AMMullan** [tenable/pyTenable#264]
**Describe the bug**
I've created a "Cloud Audit Infrastructure" scan and want to use the same scan each time an audit is required, so figure I can use tio.scans.configure(scan_id, credentials={x}) but I get an error when doing so.

**To Reproduce**
The following was run:
```python
scan_creds = {
    'Cloud Services': {
        'Amazon AWS': [{
            'access_key_id': aws_access_key_id,
            'secret_key': aws_secret_access_key
        }]
    }
}
tio.scans.configure(scan_id, credentials=scan_creds)
```

**Expected behavior**
Update credentials of scan

**Actual behavior**
Receive the following:
`PUT https://cloud.tenable.com/scans/1654 >> e2bb77bb4e17a71fd02e07da25cfd6e2:400 {"error":"A policy cannot have more than 1 set of Amazon AWS credentials."}`

**System Information (please complete the following information):**
 - OS: Ubuntu
 - Architecture 64bit

**Additional context**
Using pyTenable v1.2.6

**Comment by levwais** [comment]
TODO: follow up with scans team.

**Comment by levwais** [comment]
the current `tio.scans.configure(scan_id, credentials)` is making a `PUT /scans/{scan_id}` call with `{ 'credentials` : { 'add': {<credentials>} }}` no matter what was provided.

The solution would be that it will compare the id of the `<credentials>` provided into the function with the id of the existing `credentials` it gets back from the `GET /scans/{scan_id}` call and then decide if it should do an `add` or `edit` operation.

we can also "auto-magically" do that in the case the credentials provided are of type `Cloud Services -> Amazon AWS` since we know that it is limited to one set of credentials. 

@mvalliammal - can you please add this improvement to our backlog?

**Comment by mvalliammal** [comment]
This is solved in the PR #396, Please verify in your code as it is available in Master branch

**Comment by mvalliammal** [comment]
Done it
